js/widgets/timbre.js

```diff
@@ -407,14 +405,14 @@ function TimbreWidget() {
             ];
         }
 
-        __playLoop = function(i) {
+        __playLoop = (i) => {
             if (that._playing) {
                 that._playNote(that.notesToPlay[i][0], that.notesToPlay[i][1]);
```

Comment:
Since we are using arrow function now, we could eliminate use of `that` to save the context. Please do this across the file. Thanks

Reply:
> Since we are using arrow function now, we could eliminate use of `that` to save the context. Please do this across the file. Thanks

Okay. I will do that.